src/main/java/com/google/devtools/build/lib/remote/Chunker.java

```diff
@@ -245,18 +251,26 @@ private void maybeInitialize() throws IOException {
     if (initialized) {
       return;
     }
+    initialize(0);
+  }
+
+  private void initialize(long srcPos) throws IOException {
+    checkState(!initialized);
     checkState(data == null);
     checkState(offset == 0);
     checkState(chunkCache == null);
     try {
+      var src = dataSupplier.get();
```

Comment:
You're probably going to have to write `InputStream src = dataSupplier.get()`, since Bazel 5 requires the Java 8 language level.

Reply:
Hello @benjaminp, I tried that but still the builds are failing. Here is the [PR](https://github.com/bazelbuild/bazel/pull/15788) with the suggested changes. Could you please help us in cherry picking these changes. Thanks!